assets/app/views/editor/edit-list.js

```diff
@@ -0,0 +1,39 @@
+var fs = require('fs');
+
+var Backbone = require('backbone');
+var _ = require('underscore');
+
+var templateHtml = fs.readFileSync(__dirname + '../../templates/authenticate.html').toString();
+
+var EditorFileListView = Backbone.View.extend({
+  tagName: 'ul',
+  className: 'collection',
+  template: _.template('<li class="collection-item avatar"><a href="<%- fileUrl %>"><i class="material-icons circle"><%- type %></i><p class="title"><%- fileName %></p></a></li>'),
+  initialize: function initializeEditorFileListView(opts) {
+    this.path = opts.path;
+    this.files = opts.files || [];
+  },
+  render: function renderUserView() {
+    var self        = this,
+        owner       = this.path.owner,
+        repo        = this.path.repo,
+        branch      = this.path.branch;
+
+    if (this.files.length === 0) {
+      self.$el.append('<strong>No files in this folder</strong>');
```

Comment:
Most people don't do this and I have mixed feelings, but keeping content in JS like this makes things like localization harder in the future. Keeping string like this in a constant would help a bit.